logstash-core-event-java/src/main/java/com/logstash/Logger.java

```diff
@@ -0,0 +1,13 @@
+package com.logstash;
+
+// minimalist Logger interface to wire a logger callback in the Event class
+// for now only warn is defined because this is the only method that's required
+// in the Event class.
+// TODO: (colin) generalize this
+
+public interface Logger {
+
+    // TODO: (colin) complete interface beyond warn when needed
+
+    void warn(String message);
```

Comment:
we can, this is the minimal implementation to support what is currently used in the Event. I did not want to go through a complete logging infrastructure work at this point, but basically this is a passthrough/proxy to whatever logging strategy is used in Ruby.